tests/e2e/framework/rawVM.go

```diff
@@ -172,15 +172,13 @@ func (vm *GCPRawVM) Setup() error {
 }
 
 func buildIstioVersion() error {
-	proxy := *proxyDebTag
-	if proxy == "" {
-		currentIstioCommit, err := u.Shell("git rev-parse HEAD")
-		if err != nil {
-			return err
+	proxyURL := fmt.Sprintf(debURL, "pilot", *pilotTag)
+	if *releaseMode {
+		if *remotePath == "" {
+			return fmt.Errorf("istioctl_url cannot be empty")
 		}
-		proxy = strings.Trim(currentIstioCommit, "\n")
+		proxyURL = strings.Replace(*remotePath, "istioctl", "deb", 1)
```

Comment:
My short answer is that the replacement might need to be done for "istioctl" or "istioctl-stage", depending on when in the release it's run.

My long answer is that I was hoping that "istioctl-stage" wouldn't end up anywhere (I mostly intended it for build preservation purposes, not actual use) because it'd be more ideal to point tests at a URL to an installer tar.gz, and then the test would use just blindly and opaquely use the istioctl (and URLs from istio.VERSION) in the tar.gz.  However, there's been talk of eventually replacing updateVersion.sh with something based on helm and I don't know if or how this might affect istio.VERSION so it'd be a shame to do work that might be throwaway).  I know that post-submits don't generate a tar.gz, but if this were the only issue to using the tars all the time then I suppose post-submit could start generating them ("make istio-archive", though various parameters need to be set so that updateVersions.sh sets the appropriate values for where artifacts are stored).